configs/default/IFRC-IFLIGHT_SUCCEX_E_F4.config

```diff
@@ -13,12 +13,10 @@ resource PPM 1 A03
 resource LED_STRIP 1 B06
 resource SERIAL_TX 1 A09
 resource SERIAL_TX 2 A02
-resource SERIAL_TX 3 B10
```

Comment:
Out of interest, why is support for UART3 removed?

Reply:
Beacuse the PB10 PB11 define for UART3 and IIC2 together. In order not to cause misunderstanding, we choose to disable serial port 3.